Added FakeWebAssembleHostEnvironment, tests, and docs. (#856)
bUnit-dev/bUnit: CHANGELOG.md

### Added

> - Added `FakeWebAssemblyHostEnvironment` that implements `IWebAssemblyHostEnvironment` so that components that inject it can use this in tests. By [@KristofferStrube](https://github.com/KristofferStrube).

Reviewer: ```suggestion
- Added new test double `FakeWebAssemblyHostEnvironment` that implements `IWebAssemblyHostEnvironment`. By [@KristofferStrube](https://github.com/KristofferStrube).
```
